src/scripts/show_originals.json

```
@@ -22,6 +22,21 @@
       "type": "textarea",
       "label": "Blogs to show reblogs from (comma-separated)",
       "default": ""
+    },
+    "runOnDashboard": {
+      "type": "checkbox",
+      "label": "Run on the dashboard",
+      "default": true
+    },
+    "runOnPeepr": {
+      "type": "checkbox",
+      "label": "Run in [user friendly description of Peepr goes here]",
```

hopefully the phrase "blog view" will clearly refer to `/blog/view`

```suggestion
      "label": "Run on the blog view drawer",
```